drivers/input/Kconfig

```diff
@@ -64,13 +54,25 @@ config UINPUT_RPMSG
 	---help---
 		Enable support uinput cross core communication
 
-config UINPUT_TOUCHSCREEN
-	bool "Enable uinput touchscreen"
+config UINPUT_TOUCH
```

Comment:
Probably need to mark this with "breaking change" lable

Reply:
uinput doesn't officially release yet, so there isn't compatibility issue.